test/merge-patient-lab-iips-mocha.js

```diff
@@ -192,20 +281,6 @@ describe('merge-patient-lab-iips', function() {
             }).then(_.property('data')).should.become({id: '001',  glucose: '75',  date: '2012-02-01'});
         });
 
-        it( "should be able to call updater without sockets", function() {
```

Comment:
The description was misleading - There are no sockets in any of these tests so that description.

The functionality is still there under this test description:
it( "should have patient and labwork output state after input ports processing"